tools/README.md

```diff
@@ -112,115 +77,199 @@ export repo_owner=<your github account>
+    We pull in [BCDevOps Jenkins Pipeline Shared Lib](https://github.com/BCDevOps/jenkins-pipeline-shared-lib).  This provides us some functions for examining the Git repos and commits. We can use these functions in Jenkinsfiles and other grovy scripts.
 
-```sh
-oc -n $tools new-app -f https://raw.githubusercontent.com/BCDevOps/sonarqube/bc80961d75eed66ec70ca022a6444963341fb39f/sonarqube-postgresql-template.yaml --param=SONARQUBE_VERSION=6.7.5
-```
+* **scriptApproval.xml**
 
-Let the SonarQube pods spin up, you can then go to https://sonarqube-<$tools>.pathfinder.gov.bc.ca and watch until SonarQube is ready.  Now you can set the admin password.
+    For our configuration groovy scripts, we need to allow certain jenkins and third party plugin scripts to run.  We override the restrictions here.
 
-#### Admin Password
+* **init.groovy.d/003-create-jobs.groovy**
 
-The SonarQube server instance is created with standard insecure credentials (User: `admin`, PW:). This should be reset to something stronger and stored in an OpenShift secret so authorized developers can find it.
+    This groovy script will build 2 jobs in Jenkins.  One that will build the master branch, and one that will build pull requests.
 
-The BCDevOps SonarQube repo provides a script that will generate a random PW, set it in SonarQube, and create a secret. This can be found under the  `/provisioning` folder of the cloned BCDevOps repo.
+    To add or change the jobs, this is where you want to go.  The name of this file is important, as it needs to get run *BEFORE* the 003-register-github-webhooks.groovy included in the basic install.  Scripts are run alphabetically.  The jobs need to be created before the github webhooks are created.  Our jobs script will read secrets and configmaps created during this setup; described below.
 
-In order to directly get the password reset script, run the following:
+    These jobs are configured to use [Jenkinsfile](https://github.com/bcgov/nr-showcase-devops-tools/blob/master/Jenkinsfile) and [Jenkinsfile.cicd](https://github.com/bcgov/nr-showcase-devops-tools/blob/master/Jenkinsfile.cicd) found at the root of your project repository.  These Jenkinsfiles will make use of the OpenShift ConfigMaps we will create below.
+
+* **init.groovy.d/100-jenkins-config-set-admin-address.groovy**
+
+    This groovy script will update the admin email address.  It is not necessary, but an example of further customization to Jenkins.  If you are swiping this, keep in mind the email address is hardcoded.
+
+### Project Prerequisites
+
+We are setting up a CICD pipeline to build and deploy a specific project.  In order for this to work, there are expectations on those projects.  Currently, each project will require 3 files at the __root__ of the repository.
+
+| Name | Description |
+| --- | --- |
+| [Jenkinsfile](https://github.com/bcgov/nr-showcase-devops-tools/blob/master/Jenkinsfile) | this is the master branch build pipeline |
```

Comment:
Think we should point these at the nr-showcase-devops-sq project... must've missed that before.

Reply:
I just changed these so that they don't point to a different repository at all. I was thinking about pointing them to the example repository, but I also feel like that could be a bit misleading when jumping between so many repositories. It may be better to just say "this is the filename expected" and then the following paragraphs says the examples.

Reply:
Sounds good, makes sense